.azure/pipelines/ci.yml

```diff
@@ -683,7 +683,7 @@ stages:
   # Helix ARM64
   - template: jobs/default-build.yml
     parameters:
-      condition: ne(variables['Build.Reason'], 'PullRequest')
+      condition: and(eq(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'))
```

Comment:
I hadn't noticed how inconsistent we are about `eq` / `ne` versus `in` / `notin` for `Build.Reason` checks. I'll include this as part of dotnet/aspnetcore-internal#3540

However, my plan is to move to use `eq` and `ne` everywhere because I've never seen a build with multiple triggering event types e.g. multiple commits added to a branch result in a shift from `IndividualCI` to `BatchedCI`. Let me know if I'm missing something and there's a great reason to use `notin` here (and elsewhere in the YAML).

No need to revert that part of this change unless validation builds fail or you want to.